tests/TestCliMerge.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright (C) 2019 KeePassXC Team <team@keepassxc.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "TestCliMerge.h"
+
+#include <QTest>
+
+#include <cli/Command.h>
+#include <cli/DatabaseCommand.h>
+
+#include <cli/Merge.h>
+
+#include <QDebug>
+
+QTEST_GUILESS_MAIN(TestCliMerge)
+
+
+void TestCliMerge::testDryRunOption() {
```

Comment:
@wizzk42 I think we can move all those new tests to the `TestCli` file.

Reply:
I removed these tests entirely, they are unnecessary